content/docs/csidriver/installation/test/powermax.md

```diff
@@ -70,3 +70,89 @@ Use this procedure to perform a volume expansion test.
   - Writes some data to the PVC
   - After that, it calculates the checksum of the written data, expands the PVC, and then recalculates the checksum
   - Cleans up all the resources that were created as part of the test
+
+## Consuming existing volumes with static provisioning
+
+Use this procedure to consume existing volumes with static provisioning.
+
+1. Open your Unisphere for Powermax, and take a note of volume-id.
+2. Create PersistentVolume and use this volume-id as a volumeHandle in the manifest. Modify other parameters according to your needs.
+3. In the following example, storage class is assumed as 'powermax', cluster prefix as 'ABC' and volume's internal name as '00001', array ID as '000000000001', volume ID as '1abc23456'. The volume-handle should be in the format of `csi`-`clusterPrefix`-`volumeNamePrefix`-`id`-`arrayID`-`volumeID`.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pvol
+  namespace: test  
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 8Gi
+  csi:
+    driver: csi-powermax.dellemc.com
+    volumeHandle: csi-ABC-pmax-1abc23456-000000000001-00001
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: powermax
+  volumeMode: Filesystem
+```
+
+3. Create PersistentVolumeClaim to use this PersistentVolume.
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc
+  namespace: test
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+  storageClassName: powermax
+  volumeMode: Filesystem
+  volumeName: pvol         
+```
+
+4. Then use this PVC as a volume in a pod.
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: powermaxtest
+    namespace: test
```

Comment:
Sorry to be picky, but above you use 2 spaces for indentation and here 4 spaces.
I know the result is the same, but for sake of readability can we be consistent.

Thx

Reply:
Addressed